src/RTT_Format_Reader/RTT_Mesh_Reader.cc

```diff
@@ -48,8 +48,10 @@ void RTT_Mesh_Reader::transform2CGNS(void) {
       cell_def = Element_Definition::QUAD_4;
     else if (cell_name == "quad5")
       cell_def = Element_Definition::QUAD_5;
-    else if (cell_name == "quad6")
+    else if (cell_name == "quad6" || cell_name == "quad6a")
       cell_def = Element_Definition::QUAD_6;
+    else if (cell_name == "quad6o")
+      cell_def = Element_Definition::QUAD_6o;
```

Comment:
Please modify the unit test to cover this case (`quad6o`).  If possible, also update the unit test to cover the `quad7` case. See https://codecov.io/gh/losalamos/Draco/compare/bc928bf7bbcc6b3f34dc8a3795ef92f8c6ee385d...a6f34f96e058b86ea2f3141b86972591fcb1a76f.